tests/arc/tnewseq_legacy.nim

```diff
@@ -0,0 +1,13 @@
+discard """
+  outputsub: "(allocCount: 201, deallocCount: 201)"
```

Comment:
Are those meant to be approximations?, I can easily make code that has more alloc than dealloc,
sorry for asking, I seen the `getAllocStats` on `system` before... :slightly_smiling_face:

Reply:
No, typo.